### ISSUE TITLE
[Question] So I press "a" to add a new task?

### ISSUE THREAD

**Issue by BarbzYHOOL** [dooit-org/dooit#156]
A or a? in the menu, it is way down in the list (which is weird) and it says "add sibling" but what if i have no task? it's not a sibling?

I'm just confused

**Comment by kraanzu** [comment]
Haha. I totally get that.
If there are no tasks then `A` and `a` are similar. It's only after you add something that these two operate differently.
My thinking for that was when people want to add something, they can just press `a` and get started

**Comment by BarbzYHOOL** [comment]
maybe put it at the top of the shortcut menu, and name it "add task/subtask/..."

**Comment by kraanzu** [comment]
This now will show an error in the bar :)

If you end up trying the latest version of dooit, do lemme know your thoughts on it <3
